### ISSUE TITLE
Double-quote table names

### ISSUE THREAD

**Issue by DenisGorbachev** [numtel/pg-live-select#5]
```
2015-05-07 16:16:32 UTC [7424] LOG:  statement: DROP TRIGGER IF EXISTS postgres_channel_Avatars ON Avatars
2015-05-07 16:16:32 UTC [7424] LOG:  statement: DROP FUNCTION IF EXISTS postgres_channel_Avatars()
2015-05-07 16:16:32 UTC [7424] ERROR:  cannot drop function postgres_channel_avatars() because other objects depend on it
2015-05-07 16:16:32 UTC [7424] DETAIL:  trigger postgres_channel_Avatars on table "Avatars" depends on function postgres_channel_avatars()
```

Some weirdos like me just name their tables in CamelCase :)

**Comment by numtel** [comment]
Good catch, I think I have [added double quotes to all the places they were missing](https://github.com/numtel/pg-live-select/commit/bd29d993f458d84a85bf50f88c5c62352ce65d29) but the test suite does not support double quotes so I am not 100% sure that it is fixed.

I have released version 0.0.10 of this package and 0.0.4 of the `numtel:pg` Meteor package. Please let me know with if this new version fixes the problem.

**Comment by DenisGorbachev** [comment]
It works, thanks!
